.github/compatibility-test-matrices/release-v7.5.x/transfer-chain-a.json

```diff
@@ -0,0 +1,29 @@
+{
+  "chain-a": [
+    "release-v7.5.x"
+  ],
+  "chain-b": [
+    "v8.2.0",
+    "v7.4.0",
+    "v6.3.0",
+    "v5.4.0",
+    "v4.6.0",
+    "v3.4.0",
+    "v2.5.0",
```

Comment:
I added them because I thought it was nice to check compatibility with older versions, just in case. Even if we don't support them anymore, I thought it was good to check that we didn't break compatibility with them.